packages/config/src/handlers/forms/validation.ts

```diff
@@ -40,14 +40,12 @@ const conditional = z.object({
 /*
  * TODO: The field validation needs to be made stricter
  */
-const field = z
+const base = z
   .object({
     name: z.string(),
     type: z.string(),
```

Comment:
this does not seem to be working, as it gives error like below

`Error: A discriminator value for key `type` could not be extracted from all schema options`

seems there is no other way other than creating placeholder discriminating fields. will work on it

Reply:
done https://github.com/opencrvs/opencrvs-core/pull/7574/commits/4a182a5176bae24decb4403daa42b2d99b106434